### PR TITLE
Fix type annotation for `AIOKafkaAdminClient.create_partitions`

### LINKED ISSUE #977: invalid Type AioKafkaAdminClient  create_partitions
**Describe the bug**
there is a type error or wrong type

**Environment (please complete the following information):**
 - aiokafka version : `v0.10.0`
 - Kafka Broker version : `7.1.2-ccs (Commit:99a40ee4675ee147)`

**Reproducible example**
in `admin/client.py`
```python
    @staticmethod
    def _convert_topic_partitions(topic_partitions: Dict[str, TopicPartition]):
        return [(topic_name, (new_part.total_count, new_part.new_assignments))
                for topic_name, new_part in topic_partitions.items()]
```
while in `struct.py` `TopicPartition` defined like this 
```python
class TopicPartition(NamedTuple):
    """A topic and partition tuple"""

    topic: str
    "A topic name"

    partition: int
    "A partition id"
```
and there is no `total_count` and `new_assignments` in this type 
it must be `admin/new_partitions.py`
```python
class NewPartitions:
    """A class for new partition creation on existing topics.  Note that the
    length of new_assignments, if specified, must be the difference between the
    new total number of partitions and the existing number of partitions.
    Arguments:
        total_count (int):
            the total number of partitions that should exist on the topic
        new_assignments ([[int]]):
            an array of arrays of replica assignments for new partitions.
            If not set, broker assigns replicas per an internal algorithm.
    """

    def __init__(
        self,
        total_count,
        new_assignments=None
    ):
        self.total_count = total_count
        self.new_assignments = new_assignments

```

### PR DESCRIPTION
**Pull request by alm0ra**
Fixes #977 

### Changes

Fixes wrong typo in create_partitions() for the admin client

### Checklist

- [x] I think the code is well written
- [x] Unit tests for the changes exist
- [x] Documentation reflects the changes
- [x] Add a new news fragment into the `CHANGES` folder

### CODE REVIEW

**Comment by alm0ra** [comment]
@tvoinarovskyi @ods 

there is no documentation provided for `AIOKafkaAdminClient` so there is nothing to update and changes 
also for test, there is a test for this part [test_admin](https://github.com/aio-libs/aiokafka/blob/master/tests/test_admin.py) that satisfy this change

File: aiokafka/admin/client.py [hunk 1]
```diff
@@ -27,6 +27,7 @@
 from aiokafka.protocol.commit import GroupCoordinatorRequest, OffsetFetchRequest
 from aiokafka.protocol.metadata import MetadataRequest
 from aiokafka.structs import OffsetAndMetadata, TopicPartition
+from aiokafka.admin import NewPartitions
```
Comment (ods) [inline aiokafka/admin/client.py:30]: It would be better to avoid importing from parent package
```suggestion
from .new_partitions import NewPartitions
```

Comment (alm0ra) [reply aiokafka/admin/client.py]: @ods Already fixed and sort imports

**Comment by alm0ra** [comment]
@ods All resolved i think it's ready

**Pull request merged**